src/styles/app.scss

```diff
@@ -7,3 +7,4 @@
 @import "common";
 @import "pagination";
 @import "tabs";
+@import "swal-overrides";
```

Comment:
I would recommend adding a new `_overrides.scss` template (as I previously mentioned), then import `_swal-overrides.scss` inside it.